### PR TITLE
Add Metrilo Extension

### PR DESCRIPTION
**Pull request by nedick007**
Adding Metrilo plugin extension to the list of repositories.

### CODE REVIEW

File: satis.json [hunk 1]
```diff
@@ -142,6 +142,7 @@
       { "type": "vcs", "url": "git://github.com/netz98/N98_CustomLayoutHandles.git" },
       { "type": "vcs", "url": "git://github.com/maxbucknell/magento-amdjs.git" },
       { "type": "vcs", "url": "git://github.com/maxbucknell/foundation4-amd.git" },
+      { "type": "vcs", "url": "git://github.com/metrilo/metrilo/metrilo-magento1-integration.git" },
```
Comment (daim2k5) [inline satis.json]: Url is wrong 404

**Comment by nedick007** [comment]
Sorry for the problem. Fixed.

**Comment by nedick007** [comment]
@daim2k5 ping :)

**Comment by daim2k5** [comment]
Sorry merged

**Pull request merged**